fusesoc/section.py

```diff
@@ -364,14 +364,16 @@ class XsimSection(ToolSection):
     def __init__(self, items=None):
         super(XsimSection, self).__init__()
 
-        self._add_member('xsim_options', StringList, "Extra Xsim compile options")
```

Comment:
I will add a new parameter type: vlogdefine. Similar as vlogparam it can set a define..